### ISSUE TITLE
RSP Cover not working as per tutorial

### ISSUE THREAD

**Issue by vizzbee** [datapink/robosat.pink#71]
Going through the tutorial, I had an issue running ```osmium extract --bbox `rsp cover --extent --dir predict/images` -o osm/zanzibar.pbf osm/tanzania.pbf```. Following error occured
```
~/robosat.pink$ osmium extract --bbox `rsp cover --extent --dir predict/images` -o osm/zanzibar.pbf osm/tanzania.pbf
usage: rsp cover [-h] [--dir DIR] [--bbox BBOX] [--geojson GEOJSON]
                 [--cover COVER] [--raster RASTER] [--no_xyz] [--zoom ZOOM]
                 [--splits SPLITS]
                 out [out ...]
rsp cover: error: the following arguments are required: out
Error parsing command line: the required argument for option '--bbox' is missing
```

On separately running  `rsp cover --extent --dir predict/images OUTPUT_DIR`, i get 
```
$rsp cover --extent --dir predict/images predict/images/extent
usage: rsp [-h]  ...
rsp: error: unrecognized arguments: --extent
```

Help is much appreciated

PS:  `rsp cover -h` yields no argument --extent
```
$rsp cover -h
usage: rsp cover [-h] [--dir DIR] [--bbox BBOX] [--geojson GEOJSON]
                 [--cover COVER] [--raster RASTER] [--no_xyz] [--zoom ZOOM]
                 [--splits SPLITS]
                 out [out ...]

optional arguments:
 -h, --help         show this help message and exit

Input [one among the following is required]:
 --dir DIR          plain tiles dir path
 --bbox BBOX        a lat/lon bbox: xmin,ymin,xmax,ymax or a bbox: xmin,xmin,xmax,xmax,EPSG:xxxx
 --geojson GEOJSON  a geojson file path
 --cover COVER      a cover file path
 --raster RASTER    a raster file path

Tiles:
 --no_xyz           if set, tiles are not expected to be XYZ based.

Outputs:
 --zoom ZOOM        zoom level of tiles [required with --geojson or --bbox]
 --splits SPLITS    if set, shuffle and split in several cover subpieces. [e.g 50/15/35]
 out                cover csv output paths [required]
```

**Comment by ocourtin** [comment]
Hi @vizzbee 

Thanks for your report.

And indeed documentation was based on master, and not on previous stable 0.5.1 release.
I've just released a new 0.5.2 release to resynchronize 101 and stable release.

So make it works: `pip3 install RoboSat.pink==0.5.2`
Then follow back 101 tutorial instructions...

Thanks to test it and to report any further issue.

**Comment by vizzbee** [comment]
Thanks for your prompt reply @ocourtin
